src/MentionWrapper.js

```diff
@@ -43,20 +43,28 @@ class MentionWrapper extends Component {
     const tokens = textBeforeCaret.split(/\s/);
     const lastToken = tokens[tokens.length - 1];
 
-    // check if the text befor the caret ends with the last word
-    const triggerIdx = textBeforeCaret.endsWith(lastToken)
-      ? textBeforeCaret.length - lastToken.length
-      : -1;
+    // Get active trigger
+    let activeTrigger = 0;
+    this.triggers.forEach((trigger) => {
```

Comment:
use `triggers.some` so that you can end the loop when you find something

Reply:
Odd i thought i reviewed this already... anyways, why not just use a .find?

activeTrigger = this.triggers.find(trigger => lastToken.includes(trigger)) <- or endswith